conn-fix/README.md

```diff
@@ -0,0 +1,359 @@
+username: "user name"
+
+# The value of Password(554) tag that is used in Logon(A) message
+password: "password"
+
+# The value of the NewPassword(925) tag that is used in Logon(A) message
+newPassword: ""
+
+# If enabled the Password(554) and NewPassword(925) (if set) will value in Logon(A) message will be encrypted using RSA algorithm.
+# The key for encryption will be loaded from the file that is specified in the parameter encryptionKeyFilePath.
+encryptPassword: false
+
+# Path to the file with public key in RSA format.
+# It will be used to encrypt the passwords values, if password encryption is enabled
+encryptionKeyFilePath: ""
+
+# The value of ResetSeqNumFlag(141) tag that is used in Logon(A) message. If the value is not set , then the tag won't be sent in the Logon
+resetSeqNumFlag: true
+
+# The number of seconds to wait for a logon response before disconnecting.
+logonTimeout: 10
+
+# The number of seconds to wait for a logout response before disconnecting.
+logoutTimeout: 10
+
+# Inactivity service timeout (in seconds) in which the service will be shutdown automatically. If set to "0" - the service does not shutdown.
+idleTimeout: 0
+
+# Perform logon when service starts
+doLogonOnStart: true
+
+# The value of the tag DefaultCstmApplVerID(1408) that is used in Logon(A) message
+defaultCstmApplVerID: ""
+
+# Sender initial sequence number.
+# It is applied when service starts.
+seqNumSender: 0
+
+# The target initial sequence number.
+# It is applied when service starts.
+seqNumTarget: 0
+
+# Add NextExpectedMsgSeqNum(789) tag into Logon(A) message
+addNextExpectedMsgSeqNum: false
+
+# Ignore if the tag ResetSeqNumFlag(141) is not available in the received Logon(A) message
+ignoreAbsenceOf141tag: false
+
+# If it is set to true (checked), the service does not store messages
+performanceMode: false
+
+# Create a new session after disconnecting on the server side
+autorelogin: true
+
+# If it is specified , then we will check for OrigSendingTime in resend request
+requiresOrigSendingTime: true
+```
+
+### FIX Server
+
+The "Connect" component behaves like a server and waits for a client to connect.
+The configuration type that needs to be used is - **FIX_Server**.
+
+#### Configurable parameters
+
+The FIX Server has the following parameters to configure:
+```yaml
+# It is a required parameter. It's the FIX version that this session should use.
+# Possible values: "FIX.4.0", "FIX.4.1", "FIX.4.2", "FIX.4.3", "FIX.4.4", "FIXT.1.1"
+beginString: FIXT.1.1
+
+# It is a required parameter. Your ID that is associated with this FIX session.
+senderCompID: sender
+
+# It is a required parameter. Counter Parties ID that is associated with this FIX session.
+targetCompID: target
+
+# It is a required parameter. Directory to store sequence number and message files inside the workspace directory.
+fileStorePath: store/fix/sessions
+
+# It is a required parameter. Time of day that this FIX session becomes activated.
```

Comment:
It is a required parameter. Time of **the** day that this FIX session becomes activated.